src/main/java/com/spotify/futures/CompletableFuturesExtra.java

```diff
@@ -55,4 +60,84 @@ private CompletableFuturesExtra() {
     }
     return new CompletableToListenableFutureWrapper<V>(future);
   }
+
+  public static <T> CompletableFuture<T> immediatelyFailed(Throwable throwable) {
```

Comment:
I think 'immediatelyFailed' is too short (it doesn't sound right), why not choose the Guava name 'immediatelyFailedFuture'?